10.16.0.183 (#194)
SoftwareAG/c8y-releasenotes: content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md

---
weight: 37
title: Release 10.16.0.200
layout: bundle
---

<!--10.16.0.164-10.16.0.200-->

### Fixes

<div><table ><colgroup>
<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
<thead><tr>
<th>
Component</th>
<th>
Description</th>
<th>
Issue</th>
</tr>
</thead><tbody>

<tr>
<td>Connectivity</td>

> <td>Due to a compatibility error of the connectivity microservice with the Kite provider, an error occurred on the metrics page, when the monthly consumption data was an out-of-scope integer. This has been fixed by changing the data type.</td>

Reviewer: is it really two separate errors, or is the compatibility error showing up in the metrics page? then we should revise the entry and talk about a single error being raised

Author: Don´t get that. It is about one error. There is a compatibility error. And this leads to an error in the metric page.

Author: How about that?

Reviewer: purrfect

> <td>Due to a compatibility error of the connectivity microservice with the Kite provider, an error occurred on the metrics page, when the monthly consumption data was an out-of-scope integer. This has been fixed by changing the data type.</td>

Author: ```suggestion
<td>Due to compatibility issues of the connectivity microservice with the Kite provider, an error occurred on the metrics page, when the monthly consumption data was an out-of-scope integer. This has been fixed by changing the data type.</td>
```

<td>DM-1566</td>
</tr>

<tr>
<td>Device Management</td>
<td>When removing a device which has child devices, users can now choose either to remove a device user or to remove all child devices. Prior to this change, it was possible to choose both options in the UI, but such an action is not supported by the backend.</td>
<td>MTM-45277</td>
</tr>

<tr>
<td>Device Management</td>
<td>On bulk device registration views, the <b>Upload</b> button now stays disabled if there is no file selected or if the selected file is invalid.</td>
<td>DM-1641</td>
</tr>

<tr>
<td>Device Management</td>

> <td>On the <b>Services</b> tab in the device details, the filter for the "Status" column did not work properly and didn´t return any results when filtering was applied. This is now fixed.</td>

Reviewer: ```suggestion
<td>On the <b>Services</b> tab in the device details, the filter for the "Status" column did not work properly and didn't return results when a filter was applied. This is now fixed.</td>
```

<td>DM-2002</td>
</tr>

<tr>
<td>LWM2M</td>
<td>During the LWM2M client registration, if the agent failed to find the registered device from the database due to a communication problem, an internal object related to the device managed object was automatically removed by the agent. Afterwards, LWM2M clients always failed to connect. This is now fixed and the internal object is only removed if the actual device managed object does not exist in the database.</td>
<td>DM-2131</td>
</tr>

<tr>
<td>LWM2M</td>
<td>Due to a limitation of the Leshan library used by the LWM2M agent, only device registration messages were supported where the LWM2M version is attached to the LWM2M object. The LWM2M agent has now been extended to also accept LWM2M registration messages where the LWM2M version is attached to the LWM2M object instance.</td>
<td>DM-1893</td>
</tr>

<tr>
<td>Loriot</td>

> <td>Events created by the Loriot agent when sending uplink requests will now be considered as device requests, providing connectivity status and availability monitoring.</td>

Reviewer: ```suggestion
<td>Events created by the Loriot agent when sending uplink requests are now considered device requests, providing connectivity status and availability monitoring.</td>
```
